specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/firewallPolicy.json

```diff
@@ -1164,6 +1168,65 @@
         "modelAsString": true
       }
     },
+    "FirewallPolicyIntrusionSystemConfiguration": {
+      "description": "The operation for configuring intrusion system.",
+      "x-ms-discriminator-value": "FirewallPolicyIntrusionSystemConfiguration",
```

Comment:
I'm curious about this one. `FirewallPolicyIntrusionSystemConfiguraion` is a base class. Why do we need `x-ms-discriminator-value`?

Reply:
@MyronFanQiu you are correct, we don't need it. removed